assets/_scss/components/_disclaimer.scss

```diff
@@ -1,16 +1,33 @@
 .usa-disclaimer {
+  @include media($small-screen) {
+    font-size: $h5-font-size;
+  }
   background-color: $color-gray-lighter;
-  font-size: 1.5rem;
+  font-size: $h6-font-size;
   padding: {
     bottom: .5rem;
     top: .5rem;
   }
 }
 
+// This hides the stage note of the disclaimer in anything smaller than 690px
 .usa-disclaimer-stage {
-  @include media($medium-screen) {
+  @include media(690px) {
```

Comment:
I don't think we should be using random breakpoints. This is hard to maintain, and right now, impossible to change via variables.

Sometimes I've found things that are not "perfect" at a certain screen size. Sometimes this is better then having code that's hard to maintain.